dexmaker-mockito/build.gradle

```diff
@@ -7,8 +7,8 @@ description = "Implementation of the Mockito API for use on the Android Dalvik V
 apply plugin: 'java-library'
 apply from: "$rootDir/gradle/publishing.gradle"
 
-targetCompatibility = '1.7'
-sourceCompatibility = '1.7'
+java.targetCompatibility = '1.8'
+java.sourceCompatibility = '1.8'
```

Comment:
Just confirming for release notes - this will block adopters using java 1.7 correct?

Reply:
Ah shoot - thank you for catching this. I actually just need the `java.` bit of this statement, the `1.8` change is no longer required now that I've moved the export handling into publishing_aar. I'll push a fix in a few moments.

Reply:
I've fixed this in the new push - please take a look!